### DISCUSSION TITLE
Toggle modes

### DISCUSSION THREAD

**Discussion by sneycampos** [rurza/BatFi#83] [Ideas]
Hey! 

I was trying to figure out how can i change the "Run on battery" to the default mode (Charging to the limit) until decide to click in the "Run on battery" again to see what happens 🤣 

Maybe a better way to select the mode?

<img width="281" alt="image" src="https://github.com/user-attachments/assets/e4e6fefd-52a6-48fe-a2a8-c5690121566b">

**Comment by rurza** [comment]
Makes sense :) There is an existing "stop override" menu item that is visible in some cases. I could reuse it. What do you think? Clicking it would restore the default behavior.
